edmondatto/foodly
# Add search functionality and render results

### WHAT DOES THIS PR DO?

- Connects the application to redux and redux-saga
- Sets up action_types and action creators for the fetch recipe functionality
- Renders search results to the page

### HOW CAN THIS BE TESTED?

- Clone the repository and run `yarn install && yarn start`
- Search for a recipe e.g. cake. You should see results comprising of the recipe name, instruction details, and a thumbnail.
